From 00ccd2a0038a4e15a09096d0536a810179832466 Mon Sep 17 00:00:00 2001
From: Cosmin Tanislav <demonsingur@gmail.com>
Date: Sun, 22 Jun 2025 01:54:08 +0300
Subject: [PATCH 1/2] NubiaCamera: disable SettingHighFps

E AndroidRuntime: FATAL EXCEPTION: main
E AndroidRuntime: Process: com.android.camera, PID: 12240
E AndroidRuntime: java.lang.RuntimeException: java.lang.ClassNotFoundException: com.zte.performance.refreshrate.ScreenRefreshRateManager
E AndroidRuntime: 	at cn.nubia.camera.utils.CameraUtil.setScreenFps(CameraUtil.java:2141)
E AndroidRuntime: 	at cn.nubia.camera.fragment.MoreSetting.exitMoreSettingsFragment(MoreSetting.java:774)
E AndroidRuntime: 	at cn.nubia.camera.fragment.MoreSetting.pause(MoreSetting.java:120)
E AndroidRuntime: 	at cn.nubia.camera.fragment.MemberFragment.pauseMoreSetting(MemberFragment.java:3113)
E AndroidRuntime: 	at cn.nubia.camera.fragment.MemberFragment.onPause(MemberFragment.java:577)
E AndroidRuntime: 	at cn.nubia.camera.normal.NormalMemberFragment.onPause(NormalMemberFragment.java:397)
---
 .../nubiacameraconfig/gen/series/SERIES_NubiaZ60Ultra.smali     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/smali/cn/nubia/camera/nubiacameraconfig/gen/series/SERIES_NubiaZ60Ultra.smali b/smali/cn/nubia/camera/nubiacameraconfig/gen/series/SERIES_NubiaZ60Ultra.smali
index 455bc5d9..2b1aa0b9 100644
--- a/smali/cn/nubia/camera/nubiacameraconfig/gen/series/SERIES_NubiaZ60Ultra.smali
+++ b/smali/cn/nubia/camera/nubiacameraconfig/gen/series/SERIES_NubiaZ60Ultra.smali
@@ -312,7 +312,7 @@
     .line 63
     const-string v0, "SettingHighFps"
 
-    invoke-interface {p1, v0, v1}, Ljava/util/Map;->put(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
+    invoke-interface {p1, v0, v2}, Ljava/util/Map;->put(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
 
     .line 64
     const-string v0, "UseArcSoftBeautifyTwoInOne"
-- 
2.49.0

